spec/services/duplicate_user_finder_service_spec.rb

```diff
@@ -0,0 +1,46 @@
+describe DuplicateUserFinderService, type: :service do
+  describe ".perform" do
+    let(:user) { build(:user, first_name: "Mathieu", last_name: "Lapin", email: "lapin@beta.fr", birth_date: '21/10/2000', phone_number: '0658032518') }
+
+    subject { DuplicateUserFinderService.new(user).perform }
+
+    context "there is no other user" do
+      before { subject }
```

Comment:
this is not necessary. the line below is enough. when you write `it { xxx }` , "it" is the subject, so in that case "it" is the result of the perform method on the service instance